### PR TITLE
Update routing.rst - Troubleshooting

### PR DESCRIPTION
**Pull request by jeremy-crapet**
Hi,

A small update on the troubleshooting part, for beginners I believe it is not explicit enough.
What do you think about it?

### CODE REVIEW

**Review by jeremy-crapet — COMMENTED** [review]
Add blank line after code block (failed travis build : https://travis-ci.org/symfony/symfony-docs/builds/179251113)

File: routing.rst [hunk 1]
```diff
@@ -686,16 +686,29 @@ Here are some common errors you might see while working with routing:
...
+
+    /**
+     * @Route("/blog/show/{slug}", name="blog_show")
+     */
+    public function showAction($slug = null)
+    {
+        // ..
+    }
+
+..
```
Comment (xabbuh) [inline routing.rst]: These two dots look misplaced.

File: routing.rst [hunk 2]
```diff
@@ -686,16 +686,29 @@ Here are some common errors you might see while working with routing:
...
         // ..
     }
 
-But your route path does *not* have a ``{slug}`` wildcard (e.g. it is ``/blog/show``).
 Add a ``{slug}`` to your route path: ``/blog/show/{slug}`` or give the argument
-a default value (i.e. ``$slug = null``).
+a default value (i.e. ``$slug = null``)::
+
+    /**
+     * @Route("/blog/show/{slug}", name="blog_show")
```
Comment (xabbuh) [inline routing.rst]: IMO we should then add config examples for the other formats too for consistency.

Comment (BatsaxIV) [inline routing.rst]: For the other troubleshooting error ?

File: routing.rst (hunk 1 above)
Comment (BatsaxIV) [inline routing.rst]: They are mandatory if you wish to have a block quotation following a code example block (see : https://gist.github.com/dupuy/1855764#block-quotations "Note that in reStructuredText...")

File: routing.rst (hunk 2 above)
Comment (xabbuh) [inline routing.rst]: I mean routing examples for the YAML, XML, and PHP formats.

Comment (BatsaxIV) [inline routing.rst]: Done in the last commit, is that ok?

**Review by HeahDude — APPROVED** [review]

**Pull request closed without merging**